gst/nnstreamer/tensor_filter/tensor_filter.c

```diff
@@ -147,6 +147,29 @@ static gboolean gst_tensor_filter_sink_event (GstBaseTransform * trans,
 static gboolean gst_tensor_filter_src_event (GstBaseTransform * trans,
     GstEvent * event);
 
+/**
+ * @brief Internal data structure for tensor_filter transform.
+ */
+typedef struct _FilterTransformData
```

Comment:
suggestion to parse input and output buffer
```
TransData * parse_buffer (self, buffer, gboolean is_flexible, gboolean need_alloc)
{
  GstMemory *mem[NNS_TENSOR_SIZE_LIMIT];
  GstMapInfo info[NNS_TENSOR_SIZE_LIMIT];
  GstTensorMetaInfo meta[NNS_TENSOR_SIZE_LIMIT];
  guint num_tensors;
  gboolean is_flexible;
  gboolean allocate_in_invoke;
}

new struct for tensors to invoke
{
  GstTensorMemory in_tensors[NNS_TENSOR_SIZE_LIMIT];
  GstTensorMemory invoke_tensors[NNS_TENSOR_SIZE_LIMIT];
  GstTensorMemory out_tensors[NNS_TENSOR_SIZE_LIMIT];
}
```

Reply:
I updated data structure for tensor_filter transform. PTAL.